src/pages/profiles/[userId].tsx

```diff
@@ -258,25 +264,36 @@ function Picture({ userId, profile, updateProfile, SaveButton }: {
     [userId, profile]
   );
 
+  const updateImageParams = (x: number, y: number, zoom: number) => {
+    updateProfile('照片参数', { x, y, zoom }); 
+  };
+
   return <>
     <Heading size="md">生活照</Heading>
     <FormControl>
-      {profile.照片链接 && <Image
-        src={profile.照片链接}
-        alt="照片"
-        maxW='300px'
-        my={componentSpacing}
-      />}
-
-      {uploadToken && <>
+      <FullWidthImageSquare profile={profile} imageParams={profile.照片参数} />
+
+      {uploadToken && <Flex gap={componentSpacing}>
```

Comment:
why adding flex here? it messed up horizontal alignment of the image. the image should be left aligned.

Reply:
This is for the buttons, i remove Flex so you can see it.